modules/search/class-jetpack-search-customize.php

```diff
@@ -10,7 +10,9 @@
 	exit;
 }
 
+require_once dirname( __FILE__ ) . '/class.jetpack-search-helpers.php';
```

Comment:
Do we need the require here?

Reply:
No, although it was necessary at one point during development. I'll remove this.